common/src/main/java/us/ilite/common/io/CodexNetworkTables.java

```diff
@@ -0,0 +1,81 @@
+package us.ilite.common.io;
+
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.flybotix.hfr.codex.Codex;
+import com.flybotix.hfr.codex.CodexOf;
+import com.flybotix.hfr.util.lang.EnumUtils;
+import com.flybotix.hfr.util.log.ILog;
+import com.flybotix.hfr.util.log.Logger;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class CodexNetworkTables {
+  private final ILog mLog = Logger.createLog(CodexNetworkTables.class);
+  private Map<Integer, NetworkTable> mTables = new HashMap<>();
```

Comment:
Why not make this an enum map? It looks like the enum gets hashed, so it should be mapping the same way.